### DISCUSSION TITLE
Why does the editor conver div to p element by default?

### DISCUSSION THREAD

**Discussion by MikeAlhayek** [Alex-D/Trumbowyg#1310] [General]
Hello,

I know I can use semantic option to change the default behavior. By I am just wondering what is the reason Trumbowyg editor change `div` element to `p` element by default? Is there an advantage of using `p` instead of `div`?

**Comment by Alex-D** [comment]
Hey!
`<p>` is a paragraph, the correct semantic for a block of text: a paragraph. 
`<div>` is just a division of the HTML structure, a block of anything. There is no meaning/semantic associated with this tag.

Hope it helps :)

**Reply by MikeAlhayek** [reply]
Thank for the clarification Alex! Is there a reason why we assume that everyone using this editor will be providing a paragraph? I would think users of this editor would be creation an article/page structure of some kind not a paragraph. If only a paragraph is required, I would assume people would use simple TextArea field instead.

**Reply by Alex-D** [reply]
Because if you write an article, you will want each block of text to be a paragraph :)

**Reply by MikeAlhayek** [reply]
Is that a rule dictated by HTML? And what if you are writing a page structure? I am trying to make sense of forcing that on all users by default.
